### DISCUSSION TITLE
Relation Display Missing in 0.18.0?

### DISCUSSION THREAD

**Discussion by HenryHuevo** [pocketbase/pocketbase#3405] [Q&A] [answered]
Was this feature removed?  Not the biggest deal since functionality is not impacted, but I couldn't skim this feature being removed in a release between 0.17.6 and 0.18.0?  Was this intentional, or no?

It appears setting the "displayFields" value in the _collection manually doesn't impact this:
"displayFields":["first_name","last_name"]}}]

Updating from 0.17.6 to 0.18.0 removes the option to modify the displayFields, but on the old data file it retains the display preferences.

**Comment by ganigeorgiev** [comment, marked as answer]
It was removed in favor of the new "Presentable" field option available in each collection field to avoid the duplication of defining the same display fields for each relation (_see also https://github.com/pocketbase/pocketbase/issues/881#issuecomment-1712743627_).

**Reply by HenryHuevo** [reply]
Thanks, that definitely works!  I was toggling presentable within the relationship field rather than the thing it is referencing, hence my confusion.

As always, appreciate the speedy response and help.
